lib/controller.js

```diff
@@ -239,17 +239,28 @@ Tessel.get = function(opts) {
         }
       }
 
+
+      function finishSearchEarly(tessel) {
+        // Remove this listener because we don't need to search for the Tessel
+        seeker.removeListener('end', searchComplete);
+        // Stop searching
+        seeker.stop();
+        // Send this Tessel back to the caller
+        logAndFinish(tessel);
```

Comment:
@rwaldron does it need that even though `seeker` is within that function's scope already?